dfastmi/batch/ReporterWaqua.py

```diff
@@ -69,4 +68,4 @@ def write_report(self, output_data : OutputDataWaqua):
         DataTextFileOperations.write_simona_box(mindzb_file, output_data.data_zmin, output_data.first_min_velocity_m, output_data.first_min_velocity_n)
 
     def _get_file_location(self, output_file_name : str) -> str:
-        return self.output_dir + os.sep + ApplicationSettingsHelper.get_filename(output_file_name)
\ No newline at end of file
+        return str(self.output_dir.joinpath(ApplicationSettingsHelper.get_filename(output_file_name)))
```

Comment:
update based on outcome of dicussion with Prisca

Reply:
will update, typehint is str

Reply:
Same applies as before. Will do minimal work, after DFAST-123 we can pick this up in GridOperation Issue